docs/clone.md

```diff
@@ -0,0 +1,30 @@
+# Clone
+
+## Using `clone`
+
+```javascript
+import clone from 'consus-core/clone';
+
+let original = {
+    a: [0, 1, 2],
+    b: 'Hello, World!',
+    c: 3.14159265
+};
+
+let twin = clone(original);
+
+twin;
+/*
+{
+    a: [0, 1, 2],
+    b: 'Hello, World!',
+    c: 3.14159265
+}
+*/
+
+twin === original;
+// false
+
+twin.a === original.a;
+// false
```

Comment:
That's apparent from lines 10 and 20.

I've only added 25 and 28 to clarify that they are the same values, but different references/objects.